src/blueprint.zig

```diff
@@ -280,3 +317,8 @@ pub fn registerVoidBlock(block: Block) void {
 	voidType = block.typ;
 	std.debug.assert(voidType != 0);
 }
+
+pub fn getVoidBlock() Block {
+	std.debug.assert(voidType != null);
```

Comment:
`voidType.?` already is an assertion.
```suggestion
```
Also I think we should find a way to make this not optional, maybe even have a defined id for it (like air which is always 0). But not in this PR of course.